### PR TITLE
Update extending AD KB

### PR DESCRIPTION
**Pull request by jmalmadclc**
Added DNS entries/NTDS settings/dcdiag to article.

### CODE REVIEW

File: Disaster Recovery/safehaven-manually-mapping-disks-in-3.1.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+The sync rate command governs the speed at which writes are mirrored to the local cache or local replica, depending on the Protection Type used within a given Protection Group.
+
+Clearing existing mapping can be accomplished by running the "cleanalldiskpairs.bat" files under the SafeHaven tools menu or by removing the appropriate registry key:
+
+  HKLM->SYSTEM->CurrentControlSet->Services->DgRplct->PairList
+
+  Delete Pair X (first two keys after default are disk numbers)
+
+  ### Warning
```
Comment (jim-phillips) [inline Disaster Recovery/safehaven-manually-mapping-disks-in-3.1.md]: Need an empty line after ### Warning to get the header to show

File: Disaster Recovery/safehaven-manually mapping disks in 3.0.md [hunk 2]
```diff
@@ -11,30 +11,31 @@
...
-  Cleared disk maps can be done by running the "cleanalldiskpairs.bar" files under the SafeHaven tools menu or by removing the appropriate registery key:
+Clearing existing mapping can be accomplished by running the "cleanalldiskpairs.bat" files under the SafeHaven tools menu or by removing the appropriate registry key:
 
   HKLM->SYSTEM->CurrentControlSet->Services->DgRplct->PairList
 
   Delete Pair X (first two keys after default are disk numbers)
 
   ### Warning
-  IMPROPERLY MAPPING DISKS HAVE THE POTENTIAL TO OVERWRITE PRODUCTION DISKS WITH THE CONTENTS OF OTHER DISKS. ENSURE THAT YOUR DISK MAPPING HAS BEEN VALIDATED IN DISK MANAGER (diskmgmt.msdc) OR THAT THIS ACTION IS BEING SUPERVISED BY SAFEHAVEN ENGINEERING OR ONBOARDING ENGINEERING.
+  IMPROPERLY MAPPING DISKS HAVE THE POTENTIAL TO OVERWRITE PRODUCTION DISKS WITH THE CONTENTS OF OTHER DISKS. ENSURE THAT YOUR DISK MAPPING HAS BEEN VALIDATED IN DISK MANAGER (diskmgmt.msc) OR THAT THIS ACTION IS BEING SUPERVISED BY SAFEHAVEN ENGINEERING OR ONBOARDING ENGINEERING.
```
Comment (jim-phillips) [inline Disaster Recovery/safehaven-manually mapping disks in 3.0.md]: Add whitespace after ###

File: Disaster Recovery/safehaven-manually-mapping-disks-in-3.1.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  IMPROPERLY MAPPING DISKS HAVE THE POTENTIAL TO OVERWRITE PRODUCTION DISKS WITH THE CONTENTS OF OTHER DISKS. ENSURE THAT YOUR DISK MAPPING HAS BEEN VALIDATED IN DISK MANAGER (diskmgmt.msc) OR THAT THIS ACTION IS BEING SUPERVISED BY SAFEHAVEN ENGINEERING OR ONBOARDING ENGINEERING.
```
Comment (jim-phillips) [inline Disaster Recovery/safehaven-manually-mapping-disks-in-3.1.md]: Need whitespace after ### header

Comment (jmalmadclc) [inline Disaster Recovery/safehaven-manually-mapping-disks-in-3.1.md]: Thanks Jim, added whitespace after ### Warning in both articles.

Jacob Malmad
Sr. Cloud Architect
CenturyLink Cloud 
Mobile: 203.844.0516
Email: jake.malmad@centurylink.com<mailto:jake.malmad@centurylinkcloud.com> 
www.centurylinkcloud.com<http://www.centurylinkcloud.com/>

File: Disaster Recovery/safehaven-manually-mapping-disks-in-3.1.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+The sync rate command governs the speed at which writes are mirrored to the local cache or local replica, depending on the Protection Type used within a given Protection Group.
+
+Clearing existing mapping can be accomplished by running the "cleanalldiskpairs.bat" files under the SafeHaven tools menu or by removing the appropriate registry key:
+
+  HKLM->SYSTEM->CurrentControlSet->Services->DgRplct->PairList
+
+  Delete Pair X (first two keys after default are disk numbers)
+
+  ### Warning
```
Comment (jmalmadclc) [inline Disaster Recovery/safehaven-manually-mapping-disks-in-3.1.md]: Thanks Nathan, changes made.

Jacob Malmad
Sr. Cloud Architect
CenturyLink Cloud 
Mobile: 203.844.0516
Email: jake.malmad@centurylink.com<mailto:jake.malmad@centurylinkcloud.com> 
www.centurylinkcloud.com<http://www.centurylinkcloud.com/>

**Pull request merged**